### PR TITLE
Org details displayed

### PR DESCRIPTION
**Pull request by praveenrajendran101**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/EUI-247

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: src/organisation/containers/organisation/organisation.component.ts [hunk 1]
```diff
@@ -24,8 +24,9 @@ export class OrganisationComponent implements OnInit, OnDestroy {
 
 
   ngOnInit(): void {
-    this.organisationSubscription = this.store.pipe(select(fromStore.getOrganisationSelArr)).subscribe(( data) => {
+    this.organisationSubscription = this.store.pipe(select(fromStore.getOrganisationSelArr)).subscribe(( data: Organisation[] ) => {
       this.orgData = data;
+      console.log(data);
```
Comment (fazanki) [inline src/organisation/containers/organisation/organisation.component.ts]: please remove log

File: src/organisation/services/organisation.service.ts [hunk 2]
```diff
@@ -13,24 +14,46 @@ export const ENVIRONMENT = {
 
 @Injectable()
 export class OrganisationService {
-  constructor(private http: HttpClient) { }
+  mockOrgData: Organisation;
```
Comment (fazanki) [inline src/organisation/services/organisation.service.ts]: please add TODO with an explanation why is this here

File: src/organisation/containers/organisation/organisation.component.html [hunk 3]
```diff
@@ -3,16 +3,49 @@
...
-      <td class="govuk-table__cell">{{orgData[0]}}</td>
+      <td class="govuk-table__cell">{{orgData[0].name}}</td>
       <td class="govuk-table__cell"></td>
     </tr>
     <tr class="govuk-table__row">
       <th class="govuk-table__header" scope="row">Address</th>
       <td class="govuk-table__cell">
-        <span *ngFor="let addressLine of orgData | slice:1">{{addressLine}}<br></span>
+        <div>{{orgData[0].contactInformation.addressLine1}}</div>
+        <div>{{orgData[0].contactInformation.addressLine2}}</div>
```
Comment (fazanki) [inline src/organisation/containers/organisation/organisation.component.html]: why orgData is an array? could this be the *ngFor

Comment (praveenrajendran101) [reply src/organisation/containers/organisation/organisation.component.html]: orgData is an array but the first object orgData[0] is an object

File: src/organisation/services/organisation.service.ts (hunk 2 above)
Comment (praveenrajendran101) [reply src/organisation/services/organisation.service.ts]: This is an existing code created by you I jst changed it type from any to Organisation

File: src/organisation/containers/organisation/organisation.component.ts (hunk 1 above)
Comment (praveenrajendran101) [reply src/organisation/containers/organisation/organisation.component.ts]: this is done

File: src/organisation/services/organisation.service.ts [hunk 4]
```diff
@@ -13,24 +14,46 @@ export const ENVIRONMENT = {
...
+        dxAddress: [
+        {
+          dxNumber: 'DX 1233767890',
+          dxExchange: '091203422091208492'
+        }
+        ]
+      }
+    }
+    );
+    console.log(this.mockOrgData);
```
Comment (adnanakgun) [inline src/organisation/services/organisation.service.ts]: console.log here.

**Comment by adnanakgun** [comment]
Only one minor suggestion. Looks good.

File: src/organisation/services/organisation.service.ts (hunk 4 above)
Comment (praveenrajendran101) [reply src/organisation/services/organisation.service.ts]: Good catch. This is done

**Comment by Vamshi-Muniganti** [comment]
Not a relevant pr to this repo, confirmed by Praveen and Mo
